README.md

```diff
@@ -52,24 +43,30 @@ Each page in this project must comply with the following layout requirements:
 
 ### Required Content
 
-- Uses at least one CSS file (likely named `style.css`)
+In this wave, we will add CSS styling to change the appearance of the site.
+
+Create and use one main CSS document named `style.css`, which will hold styles that apply to every page. Then create three additional CSS documents for each HTML page's specific styles, named `index.css`, `facts.css`, and `gallery.css`.
+
+**Each html page should link to to two style sheets:** `style.css` first, and then CSS document that corresponds to it (`index.css`, `facts.css`, or `gallery.css`). This will ensure that the specific styles override the generic one.
```

Comment:
```suggestion
**Each html page should link to two style sheets:** `style.css` first, and then CSS document that corresponds to it (`index.css`, `facts.css`, or `gallery.css`). This will ensure that the specific styles override the generic one.
```